### DISCUSSION TITLE
The location Icon doesn't show on GitHub Pages

### DISCUSSION THREAD

**Discussion by torn4dom4n** [LekoArts/gatsby-themes#1056] [Questions] [answered]
I have 2 sites built from [this repo](https://github.com/area44-stack/nganguyen) but GitHub Pages doesn't show the location icon. [Netlify Sites](https://nganguyenphotos.netlify.app) and [GitHub Pages](https://area44-stack.github.io/nganguyen):

![image](https://user-images.githubusercontent.com/27698189/202911885-3eaaa9c8-dfe5-453f-81cf-f02c092f14df.png)

**Comment by torn4dom4n** [comment]
After I used Develop Tool, I found this:

![image](https://user-images.githubusercontent.com/27698189/202918652-a3f8f7ab-109a-4c8f-b860-a26091cfa63c.png)

**Comment by LekoArts** [comment, marked as answer]
This should be fixed with https://github.com/LekoArts/gatsby-themes/pull/1057 - can you update your version of the theme and try again?

**Reply by torn4dom4n** [reply]
Thank you. It worked.
